README.md

```diff
@@ -235,15 +235,17 @@ jobs:
       - name: Test action step PAT
         uses: AndreasAugustin/actions-template-sync@v2
         with:
-          github_token: ${{ secrets.CUSTOM_GITHUB_PAT }}
+          source_gh_token: ${{ secrets.CUSTOM_GITHUB_PAT }}
           source_repo_path: ${{ secrets.SOURCE_REPO_PATH }} # <owner/repo>, should be within secrets
 ```
 
 ### Action Inputs
 
 | Variable                    | Description                                                                                                   | Required | Default                                                           |
 |-----------------------------|---------------------------------------------------------------------------------------------------------------|----------|-----------------------------------------------------------------------|
-| github_token                | Token for the repo. Can be passed in using `${{ secrets.GITHUB_TOKEN }}`                                     | `true`   |   `${{ github.token }}`                                                                    |
+| github_token                | :warning: [Deprecated] please use `source_gh_token` instead to have a declarative name. Token for the repo. Can be passed in using `${{ secrets.GITHUB_TOKEN }}`                                     | `true`   |   `${{ github.token }}`                                                                    |
```

Comment:
deprecation notice for the `github_token` variable. Is also mentioned within the action.yml.

Reply:
This is a very good idea :+1: